src/modules/food.js

```diff
@@ -1,2 +1,3 @@
 /* eslint-disable */
```

Comment:
Same issue as discussed above. Please enable linters and try to resolve the issue.

Reply:
Thank you so much and everything is resolved.